packages/foreman/foreman/foreman.spec

```diff
@@ -944,8 +944,8 @@ Meta Package to install requirements for telemetry support
 Summary: Foreman journald logging support
 Group:  Applications/System
 # start specfile journald Requires
-Requires: %{?scl_prefix}rubygem(logging-journald) >= 1.0
-Requires: %{?scl_prefix}rubygem(logging-journald) < 2.0
+Requires: %{?scl_prefix}rubygem(logging-journald) >= 2.0
```

Comment:
It'd be nice if in this would also be updated in https://github.com/theforeman/foreman/blob/7673103014073bf01908b5e7252956d5b35d4efa/bundler.d/journald.rb#L3 though bundler_ext does ignore this part.

Reply:
Will do core PR with that.

Reply:
https://github.com/theforeman/foreman/pull/6145